daemon/tests/harness/flow.rs

```diff
@@ -55,9 +58,12 @@ where
     T: Clone,
 {
     rx.changed()
-        .timeout(Duration::from_secs(120))
+        .timeout(NEXT_WAIT_TIME)
```

Comment:
Would be nice to undo the original patch then if we overwrite it with a constant here.